src/Bicep.Cli/Services/ParamsFileWriter.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Emit;
+using Bicep.Core.Exceptions;
+using Bicep.Core.Syntax;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+
+namespace Bicep.Cli.Services
+{
+    public class ParamsFileWriter
+    {
+        private readonly InvocationContext invocationContext;
+
+        public ParamsFileWriter(InvocationContext invocationContext)
+        {
+            this.invocationContext = invocationContext;
+        }
+
+        public EmitResult ToFile(ProgramSyntax syntax, string outputPath)
+        {
+            using var fileStream = CreateFileStream(outputPath);
+            return new ParamsEmitter(syntax, invocationContext.EmitterSettings).EmitParamsFile(fileStream);
+        }
+
+        public EmitResult ToStdout(ProgramSyntax syntax)
+        {
+            using var writer = new JsonTextWriter(invocationContext.OutputWriter)
+            {
+                Formatting = Formatting.Indented
+            };new ParamsEmitter(syntax, invocationContext.EmitterSettings).EmitParamsFile(writer);
```

Comment:
Is this line intentional?

Reply:
No, I haven't looked in adding support for stdout for params yet so I will update this